<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditTaskDialog</class>
 <widget class="QDialog" name="EditTaskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>207</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="titleEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="descriptionEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dueDateLabel">
         <property name="text">
          <string>Due date</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="dueDateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
